sklearn/plot/partial_dependence.py

```diff
@@ -0,0 +1,325 @@
+                'target must be in [0, n_tasks], got {}.'.format(target))
+        target_idx = target
+    else:
+        target_idx = 0
+
+    # get global min and max values of PD grouped by plot type
+    pdp_lim = {}
+    for pd, values in pd_result:
+        min_pd, max_pd = pd[target_idx].min(), pd[target_idx].max()
+        n_fx = len(values)
+        old_min_pd, old_max_pd = pdp_lim.get(n_fx, (min_pd, max_pd))
+        min_pd = min(min_pd, old_min_pd)
+        max_pd = max(max_pd, old_max_pd)
+        pdp_lim[n_fx] = (min_pd, max_pd)
+
+    # create contour levels for two-way plots
+    if 2 in pdp_lim:
+        Z_level = np.linspace(*pdp_lim[2], num=8)
+
+    if fig is None:
+        fig = plt.figure(**fig_kw)
+    else:
+        fig.clear()
+
+    if line_kw is None:
+        line_kw = {'color': 'green'}
+    if contour_kw is None:
+        contour_kw = {}
+
+    n_cols = min(n_cols, len(features))
+    n_rows = int(np.ceil(len(features) / float(n_cols)))
+    axs = []
+    for i, fx, name, (pd, values) in zip(count(), features, names, pd_result):
+        ax = fig.add_subplot(n_rows, n_cols, i + 1)
+
+        if len(values) == 1:
+            ax.plot(values[0], pd[target_idx].ravel(), **line_kw)
+        else:
+            # make contour plot
+            assert len(values) == 2
+            XX, YY = np.meshgrid(values[0], values[1])
+            Z = pd[target_idx].T
+            CS = ax.contour(XX, YY, Z, levels=Z_level, linewidths=0.5,
+                            colors='k')
+            ax.contourf(XX, YY, Z, levels=Z_level, vmax=Z_level[-1],
+                        vmin=Z_level[0], alpha=0.75, **contour_kw)
+            ax.clabel(CS, fmt='%2.2f', colors='k', fontsize=10, inline=True)
+
+        # plot data deciles + axes labels
+        deciles = mquantiles(X[:, fx[0]], prob=np.arange(0.1, 1.0, 0.1))
+        trans = transforms.blended_transform_factory(ax.transData,
+                                                     ax.transAxes)
+        ylim = ax.get_ylim()
+        ax.vlines(deciles, [0], 0.05, transform=trans, color='k')
+        ax.set_xlabel(name[0])
+        ax.set_ylim(ylim)
+
+        # prevent x-axis ticks from overlapping
+        ax.xaxis.set_major_locator(MaxNLocator(nbins=6, prune='lower'))
+        tick_formatter = ScalarFormatter()
+        tick_formatter.set_powerlimits((-3, 4))
+        ax.xaxis.set_major_formatter(tick_formatter)
+
+        if len(values) > 1:
+            # two-way PDP - y-axis deciles + labels
+            deciles = mquantiles(X[:, fx[1]], prob=np.arange(0.1, 1.0, 0.1))
+            trans = transforms.blended_transform_factory(ax.transAxes,
+                                                         ax.transData)
+            xlim = ax.get_xlim()
+            ax.hlines(deciles, [0], 0.05, transform=trans, color='k')
+            ax.set_ylabel(name[1])
+            # hline erases xlim
+            ax.set_xlim(xlim)
+        else:
+            ax.set_ylabel('Partial dependence')
```

Comment:
I think that giving the name of the target, in this case, will be much better than `Partial dependence`.

Reply:
I don't think we have access to the target name